app/models/vacancy.rb

```diff
@@ -27,6 +28,8 @@ class Vacancy < ApplicationRecord
     [I18n.t('jobs.sort_by.expiry_time.ascending'), 'expiry_time_asc']
   ]
 
+  SUBJECT_OPTIONS = Subjects::SUBJECT_OPTIONS.freeze
```

Comment:
`include Subjects` is a little terser and ensures that `SUBJECT_OPTIONS` is always referencing the same object.